pkg/cmd/cli/cmd/newapp.go

```diff
@@ -144,6 +144,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 	cmd.Flags().BoolVar(&config.InsecureRegistry, "insecure-registry", false, "If true, indicates that the referenced Docker images are on insecure registries and should bypass certificate checking")
 	cmd.Flags().BoolVarP(&config.AsList, "list", "L", false, "List all local templates and image streams that can be used to create.")
 	cmd.Flags().BoolVarP(&config.AsSearch, "search", "S", false, "Search all templates, image streams, and Docker images that match the arguments provided.")
+	cmd.Flags().BoolVar(&config.AllowMissing, "allow-missing", false, "If true, indicates that referenced Docker images that cannot be found locally or in a registry should still be used.")
```

Comment:
Rule of thumb is going to be how often someone needs this, and the rarer it
is the longer the description is allowed to be.  I assume this is only in
very specialized cases, like I'm going to be pushing an image and I want to
create it later?  That seems in the 1-2% of the time range.

On Sun, Oct 4, 2015 at 8:47 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/cmd/cli/cmd/newapp.go
> <https://github.com/openshift/origin/pull/4891#discussion_r41100831>:
>
> > @@ -144,6 +144,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
> >  	cmd.Flags().BoolVar(&config.InsecureRegistry, "insecure-registry", false, "If true, indicates that the referenced Docker images are on insecure registries and should bypass certificate checking")
> >  	cmd.Flags().BoolVarP(&config.AsList, "list", "L", false, "List all local templates and image streams that can be used to create.")
> >  	cmd.Flags().BoolVarP(&config.AsSearch, "search", "S", false, "Search all templates, image streams, and Docker images that match the arguments provided.")
> > +	cmd.Flags().BoolVar(&config.AllowMissing, "allow-missing", false, "If true, indicates that referenced Docker images that cannot be found locally or in a registry should still be used.")
>
> (i'm not against allow-missing-images, just felt really long)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4891/files#r41100831>.
>